love/src/components/NightReport/CreateNightReport.jsx

```diff
@@ -53,21 +53,22 @@ function ProgressBarSection({ currentStep, currentStatusText }) {
   );
 }
 
-function ObserversField({ isEditDisabled, observersFieldRef, userOptions, selectedUsers, setSelectedUsers }) {
+function ObserversField({ isEditDisabled, userOptions, selectedUsers, setSelectedUsers }) {
+  const memoizedSelectedValueDecorator = useCallback(
+    (v) => (v.length > MULTI_SELECT_OPTION_LENGHT ? `...${v.slice(-MULTI_SELECT_OPTION_LENGHT)}` : v),
```

Comment:
Length has a typo here. Might be good to fix for better maintenance.

Reply:
thanks for catching that! I'll make the change :+1:.